README.md

```diff
@@ -97,6 +97,44 @@ We use Code generation to generate our types for our GraphQL queries and mutatio
 - Each query and mutation should have a unique name.
 - Since query analysis for type generation occurs statically we cant place dynamic variables with in query strings we instead have to hard code the variable in the query or pass it in as query variable.
 
+## How to get data for your feature
+If you need more data to be able to test out your feature locally the easiest way to do it is to populate the local db using real data from the staging or production environments.
+
+1. You should identify if the data you need is located in the staging or prod db and ssh into them (You should be connected to the office network or vpn before proceeding). The urls for these db servers can be located in the `fabfile.py` located in the evergreen directory or [here](https://github.com/10gen/kernel-tools/blob/master/evergreen/fabfile.py).
+2. You should ensure you are connected to a secondary node before proceeding.
+3. Run `mongo` to open the the mongo shell.
+4. Identify the query you need to fetch the data you are looking for. 
+
+    ```
+    mci:SECONDARY> rs.slaveOk() // Allows read operations on a secondary node
```

Comment:
just flagging the casual racism in mongo terminology 🏴

Reply:
I believe Theres an effort on the way to eliminate it or there at least should be.